config/settings.yml

```diff
@@ -6472,3 +6572,271 @@ loadout:
+      - weapon: aim9l
+      - weapon: aim9m
+      - weapon: cap9m
+      - weapon: asqt50
+    1b:  &f14station1b
+      - weapon: aim54am47
+      - weapon: aim54am60      
+      - weapon: aim54cm47
+      - weapon: aim9l
+      - weapon: aim9m
+      - weapon: aim7m
+        pylon: lau115
+      - weapon: aim7f
+        pylon: lau115
+      - weapon: aim7mh
+        pylon: lau115
+      - weapon: mk20
+        amount: 2
+      - weapon: mk81
+        amount: 2
+      - weapon: mk82
+        amount: 2
+      - weapon: mk82se
+        amount: 2
+      - weapon: mk82air
+        amount: 2
+      - weapon: mk83
+      - weapon: bdu33
+        amount: 3    
+      - weapon: lau10mk71  
+    2: &f14station2
+      - weapon: ft300
+      - weapon: ft300ae
+    3: &f14station3
+      - weapon: aim54am47
+      - weapon: aim54am60      
+      - weapon: aim54cm47
+      - weapon: aim7m
+        pylon: lau115
+      - weapon: aim7f
+        pylon: lau115
+      - weapon: aim7mh
+        pylon: lau115
+      - weapon: adm141a
+      - weapon: gbu10
+      - weapon: gbu12
+      - weapon: gbu16
+      - weapon: gbu24
+      - weapon: lau10mk71    
+      - weapon: mk81
+        amount: 4
+      - weapon: mk82
+      - weapon: mk82
+        amount: 4
+      - weapon: mk82se
+      - weapon: mk82se
+        amount: 4
+      - weapon: mk82air
+      - weapon: mk82air
+        amount: 4
+      - weapon: mk83
+      - weapon: mk20
+      - weapon: mk84
+      - weapon: bdu33
+        amount: 3
+    4: &f14station4
+      - weapon: aim54am47
+      - weapon: aim54am60      
+      - weapon: aim54cm47
+      - weapon: aim7m
+        pylon: lau115
+      - weapon: aim7f
+        pylon: lau115
+      - weapon: aim7mh
+        pylon: lau115
+      - weapon: adm141a
+      - weapon: gbu12
+      - weapon: gbu16
+      - weapon: suuluu    
+      - weapon: mk81
+        amount: 3
+      - weapon: mk20
+      - weapon: mk84
+      - weapon: mk82
+      - weapon: mk82
+        amount: 3
+      - weapon: mk82se
+      - weapon: mk82se
+        amount: 3
+      - weapon: mk82air
+      - weapon: mk82air
+        amount: 3
+      - weapon: mk83
+      - weapon: bdu33
+        amount: 3
+    5: &f14station5
+      - weapon: aim54am47
+      - weapon: aim54am60      
+      - weapon: aim54cm47
+      - weapon: aim7m
+        pylon: lau115
+      - weapon: aim7f
+        pylon: lau115
+      - weapon: aim7mh
+        pylon: lau115
+      - weapon: adm141a
+      - weapon: gbu12
+      - weapon: gbu16
+      - weapon: gbu24
+      - weapon: suuluu    
+      - weapon: mk81
+        amount: 3
+      - weapon: mk20
+      - weapon: mk84
+      - weapon: mk82
+      - weapon: mk82
+        amount: 3
+      - weapon: mk82se
+      - weapon: mk82se
+        amount: 3
+      - weapon: mk82air
+      - weapon: mk82air
+        amount: 3
+      - weapon: mk83
+      - weapon: bdu33
+        amount: 3
+    6: *f14station3
+    7: *f14station2
+    8a: &f14station8b
```

Comment:
key is 8a, but anchor is 8b?

Also please remove the anchors where they are not needed (as they are not referenced someplace else).

Reply:
I removed anchor &f14station8b